doc/content/guides/connecting-gateways/multitechconduit/.editorconfig

```diff
@@ -0,0 +1,24 @@
+root = true
```

Comment:
Remove this file

Reply:
This entire file is unnecessary. We already have one in the root of the repo.